icevision/models/mmdet/download_configs.py

```diff
@@ -3,7 +3,7 @@
 from icevision.imports import *
 from icevision.utils import *
 
-VERSION = "v2.16.0"
+VERSION = "v2.20.0"
```

Comment:
Can you please change version to : `VERSION = "v2.20.1"

See https://github.com/airctic/mmdetection_configs/releases/tag/v2.20.1

I added `1` to differentiate from the official `mmdetection` version since it's a slightly different version.

Reply:
Done!